concordia/views.py

```diff
@@ -50,6 +51,7 @@
 ASSETS_PER_PAGE = 36
 PROJECTS_PER_PAGE = 36
 ITEMS_PER_PAGE = 36
+URL_REGEX = "http[s]?://"
```

Comment:
I try to always prefix regular expressions with `r` to mark them as [raw string literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals). In addition to not needing to escape back-slashes, in many editors and on GitHub that activates regex syntax highlighting:

```suggestion
URL_REGEX = r"http[s]?://"
```

Reply:
I meant to do that, but I forgot